backend/src/main/java/com/example/demo/todo/TodoService.java

```diff
@@ -27,11 +27,15 @@ public Optional<Todo> findById(String id) {
 
     public Optional<Todo> updateTodo(String id, String email, Todo updatedTodo) {
                 Optional<Todo> todoFromDB = todoRepository.findByIdAndUserEmail(id, email);
-                // abfrafe zum verglichen kommt hier
-                        return
+                int currentScore = 0;
+                if(updatedTodo.getStatus() != todoFromDB.get().getStatus()){
+                } currentScore++;
+                updatedTodo.setScore(currentScore);
                 todoFromDB.map(todo -> todo.update(updatedTodo))
                 .map(todoRepository::save);
```

Comment:
wahrscheinlich soll alles bis hier in den if oder?

Reply:
nein. erst wenn score gesetzt wird, dann bekomme ich eine updated todo